doc/classes/EditorSettings.xml

```diff
@@ -408,9 +479,23 @@
 		<member name="editors/animation/onion_layers_past_color" type="Color" setter="" getter="">
 			The modulate color to use for "past" frames displayed in the animation editor's onion skinning feature.
 		</member>
+		<member name="editors/bone_mapper/handle_colors/error" type="Color" setter="" getter="">
+		</member>
+		<member name="editors/bone_mapper/handle_colors/missing" type="Color" setter="" getter="">
+		</member>
+		<member name="editors/bone_mapper/handle_colors/set" type="Color" setter="" getter="">
+		</member>
+		<member name="editors/bone_mapper/handle_colors/unset" type="Color" setter="" getter="">
+		</member>
+		<member name="editors/grid_map/editor_side" type="int" setter="" getter="">
+		</member>
+		<member name="editors/grid_map/palette_min_width" type="int" setter="" getter="">
+		</member>
```

Comment:
```suggestion
		<member name="editors/grid_map/palette_min_width" type="int" setter="" getter="">
		</member>
```
```suggestion
		<member name="editors/grid_map/palette_min_width" type="int" setter="" getter="">
			Minimum width of GridMap's mesh palette side panel.
		</member>
```
Seems like it requires restarting the editor, because it's not handled correctly.